packages/web/src/features/translation/TranslationView.tsx

```diff
@@ -146,6 +143,15 @@ export default function TranslationView() {
           );
         })}
       </ol>
+    );
+  }
+  return (
+    <div className="px-4 flex flex-grow flex-col gap-2">
+      <VerseSelector
+        verseId={params.verseId}
+        onVerseChange={(verseId) => navigate('../translate/' + verseId)}
+      />
+      {content}
```

Comment:
One trick I use when I a need a large conditional in the middle of a template like this is to use a function that is immediately called. This moves the conditional to the template instead of having to declare new variables and if statements outside the template. Up to you if you want to refactor this. I don't have a strong preference
```tsx
<parent>
{
  (() => {
    if (loading) {
      return <span />
    } else {
      return <div />
    }
  })()
}
</parent>
```

Reply:
Done